### PR TITLE
Mx remove negative validation

### PR DESCRIPTION
**Pull request by samlown**

### CODE REVIEW

File: regimes/mx/invoice.go [hunk 1]
```diff
@@ -118,32 +110,20 @@ func (v *invoiceValidator) validSupplier(value interface{}) error {
...
 
-func (v *invoiceValidator) validLine(value interface{}) error {
-	line, _ := value.(*bill.Line)
-	if line == nil {
-		return nil
-	}
-
-	return validation.ValidateStruct(line,
-		validation.Field(&line.Quantity, num.Positive),
-		validation.Field(&line.Total, num.Positive),
```
[the remark this answers is not in the document]
Comment (cavalle) [reply regimes/mx/invoice.go:129]: I confirmed that:
* When quantity greater than zero and the total is zero, and there's no tax information the PAC accepts the CFDI
* When the quantity is zero or lower, the PAC returns an error: `{"message":"CFDI40999 - Error no clasificado.","messageDetail":"The 'Cantidad' attribute is invalid - The value '0' is invalid according to its datatype 'Decimal' - The MinInclusive constraint failed.","data":null,"status":"error"}`
* When the total is lower than zero, the PAC returns an error: `{"message":"CFDI40999 - Error no clasificado.","messageDetail":"The 'Total' attribute is invalid - The value '-50.00' is invalid according to its datatype 'http://www.sat.gob.mx/sitio_internet/cfd/tipoDatos/tdCFDI:t_Importe' - The Pattern constraint failed.","data":null,"status":"error"}`

In summary, I think we should keep validating each invoice line like this:

```go
return validation.ValidateStruct(line,
    validation.Field(&line.Quantity, num.Positive),
    validation.Field(&line.Total, num.Min(num.MakeAmount(0, 0))),
)
```

[the remark this answers is not in the document]
Comment (samlown) [reply regimes/mx/invoice.go:129]: Okay, I have really mixed feelings about this, but I think the proposal makes sense. I'll re-add the line validation with a minimum check.

**Review by cavalle — APPROVED** [review]
LGTM!
